apps/windows/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "FluentTester-windows",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint .",
+    "windows": "react-native run-windows",
+    "bundle": "fluentui-scripts metro --cli"
+  },
+  "dependencies": {
+    "@fluentui-react-native/tester": "^0.0.1",
+    "@uifabricshared/theming-react-native": "0.7.0",
+    "react": "~16.11.0",
+    "react-native": "0.62.2",
+    "react-native-windows": "^0.0.0-master.45"
```

Comment:
As stated, this will also bring in 0.63, 0.64, etc as we merge into our master.

Current plan is to release 0.62.0-preview.1 on Monday if you can wait.

Reply:
Because the experience of trying to build UWP was so horrible for people on the 0.60 versions if we can get folks unblocked I'd probably check this in and then just switch off master once the preview version is up.